src/packages.jl

```diff
@@ -5,9 +5,11 @@ import Downloads
 import Pidfile
 import Pkg
 import ...GAP: disable_error_handler, Globals, GapObj, replace_global!, RNamObj, sysinfo, Wrappers
-import ...GAP: gap_pkg_jlls, Compat, Setup
+import ...GAP: gap_pkg_jlls, Compat, Setup, GAP_VERSION, GAP_jll, GAP_lib_jll
 
-const DEFAULT_PKGDIR = Ref{String}()
+gap_pkgdir() = joinpath(Base.DEPOT_PATH[1], "gaproot", "v$(GAP_VERSION.major).$(GAP_VERSION.minor)")
```

Comment:
I find it a bit confusing that this function is called `pkgdir` although it contains the path to a GAP root path and *not* to a GAP packagedir.

Reply:
This is still the same issue for me

Reply:
changed it, please have another look

Reply:
The new name is less confusing, thanks!